src/table/cells/handle-cell-focus.js

```diff
@@ -94,3 +94,7 @@ export const handleFocusoutEvent = (evt, shouldRefocus, blur) => {
     blur(false);
   }
 };
+
+// get the object, find the patent of the confirmbutton in slection toolbar parent and focus that element
+export const focusConfirmButton = (element) =>
+  element.closest('.qv-object-wrapper')?.querySelector('.sel-toolbar-confirm')?.parentElement?.focus();
```

Comment:
only valid if client?
So nothing outside right now, right?

Reply:
the normal tab events will be canceled everywhere. We can add a `keyboard.enabled` check for now, or just don't care that it breaks outside the client for now